lang/cs/Org.Apache.REEF.Client/YARN/WindowsYarnJobCommandBuilder.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System.Text;
+using Org.Apache.REEF.Client.Yarn;
+using Org.Apache.REEF.Client.YARN.Parameters;
+using Org.Apache.REEF.Common.Files;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Client.YARN
+{
+    /// <summary>
+    /// WindowsYarnJobCommandBuilder is .NET implementation of `org.apache.reef.runtime.common.launch.JavaLaunchCommandBuilder`
+    /// This class provides the command to be submitted to RM for execution of .NET driver running on Windows environment.
+    /// </summary>
+    internal sealed class WindowsYarnJobCommandBuilder : IYarnJobCommandBuilder
```

Comment:
Our aim from the beginning has been to move entire .NET REEF client for all current environments it works on to the new YARN REST API based client. There are two separate JIRA items we are tackling (successfully so far) REEF-70 and REEF-227.

To answer your question, this is for any Windows cluster that we currently use .NET REEF client on.